test/e2e/e2e_topologyspreadconstraint_test.go

```diff
@@ -181,9 +186,6 @@ func TestTopologySpreadConstraint(t *testing.T) {
 			plugin.(frameworktypes.BalancePlugin).Balance(ctx, workerNodes)
 			t.Logf("Finished RemovePodsViolatingTopologySpreadConstraint strategy for %s", name)
 
-			t.Logf("Wait for terminating pods of %s to disappear", name)
-			waitForTerminatingPodsToDisappear(ctx, t, clientSet, deployment.Namespace)
```

Comment:
This is needed for synchronization so new pods get a chance to be started

Reply:
We are currently running the descheduler inside a container, so we just need to continuously wait for the results using `wait.PollUntilContextTimeout`. We don’t actually know when the descheduler evicts the pods.
https://github.com/kubernetes-sigs/descheduler/blob/5464f377a10f217fe4c35a756fcdf43ca680a7a2/test/e2e/e2e_topologyspreadconstraint_test.go#L258

Reply:
> We don’t actually know when the descheduler evicts the pods

We do as we expect the descheduler to evict certain pods in a certain time based on the configuration the test sets.